### PR TITLE
Update scraping-service.md

### PR DESCRIPTION
**Pull request by annelaurefroment**
Fixes # grafana/support-escalations#6830

I'm adding information on the /debug/ring endpoint.  This is based on this PR: https://github.com/grafana/agent/pull/4577 which had been abandoned.

I'm only backporting back to 0.37. Afaik we only support 2 versions back, but this was added in 0.33

### CODE REVIEW

**Comment by annelaurefroment** [comment]
@clayton-cornell I see some checks are failing but I have no idea what could be causing that. Is there anything I need to change?

File: docs/sources/static/configuration/scraping-service.md [hunk 1]
```diff
@@ -168,6 +168,24 @@ container with the `grafana/agentctl` image. Tanka configurations that
...
+
+- The list of Agents in the cluster along with their respective tokens
+  used for sharding.
+- The list of configuration files in the KV store and their associated hash values used for lookup in the ring.
+- The unique instance ID assigned to each instance of the Agent running in the cluster. The instance ID is a unique identifier assigned to each running instance of the Agent within the cluster.
+The exact details of the instance ID generation might be specific to the implementation of the Grafana Agent.
+- The time of the "Last Heartbeat" of each instance. The Last Heartbeat is the last time the instance was active in the ring.
+
+
+
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/scraping-service.md:188]:
```suggestion
- The time of the "Last Heartbeat" of each instance. The Last Heartbeat is the last time the instance was active in the ring.

```

Removing blank lines

File: docs/sources/static/configuration/scraping-service.md [hunk 2]
```diff
@@ -168,6 +168,24 @@ container with the `grafana/agentctl` image. Tanka configurations that
...
+You can access this endpoint by making an HTTP request to the Agent's API server.
+
+Information returned by the `/debug/ring` endpoint includes:
+
+- The list of Agents in the cluster along with their respective tokens
+  used for sharding.
+- The list of configuration files in the KV store and their associated hash values used for lookup in the ring.
+- The unique instance ID assigned to each instance of the Agent running in the cluster. The instance ID is a unique identifier assigned to each running instance of the Agent within the cluster.
+The exact details of the instance ID generation might be specific to the implementation of the Grafana Agent.
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/scraping-service.md:184]:
```suggestion
- The unique instance ID assigned to each instance of the Agent running in the cluster.
   The instance ID is a unique identifier assigned to each running instance of the Agent within the cluster.
   The exact details of the instance ID generation might be specific to the implementation of the Grafana Agent.
```

Semanitic line breaks and ensuring that the sentences in teh bullet remain attached to the bullet.

File: docs/sources/static/configuration/scraping-service.md [hunk 3]
```diff
@@ -168,6 +168,24 @@ container with the `grafana/agentctl` image. Tanka configurations that
 utilize `grafana/agentctl` and sync a set of configurations to the API
 are planned for the future.
 
+## Debug Ring endpoint
+
+You can use the `/debug/ring` endpoint to troubleshoot issues with the scraping service in Scraping Service Mode. 
+It provides information about the Distributed Hash Ring and the current distribution of configurations among Agents in the cluster and gives you the ability to forget an instance in the ring manually.
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/scraping-service.md:174]:
```suggestion
It provides information about the Distributed Hash Ring and the current distribution of configurations among Agents in the cluster.
It also allows you to forget an instance in the ring manually.
```

File: docs/sources/static/configuration/scraping-service.md [hunk 4]
```diff
@@ -168,6 +168,24 @@ container with the `grafana/agentctl` image. Tanka configurations that
...
+
+You can use the `/debug/ring` endpoint to troubleshoot issues with the scraping service in Scraping Service Mode. 
+It provides information about the Distributed Hash Ring and the current distribution of configurations among Agents in the cluster and gives you the ability to forget an instance in the ring manually.   
+
+You can access this endpoint by making an HTTP request to the Agent's API server.
+
+Information returned by the `/debug/ring` endpoint includes:
+
+- The list of Agents in the cluster along with their respective tokens
+  used for sharding.
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/scraping-service.md:181]:
```suggestion
- The list of Agents in the cluster, and their respective tokens used for sharding.
```

File: docs/sources/static/configuration/scraping-service.md [hunk 5]
```diff
@@ -168,6 +168,24 @@ container with the `grafana/agentctl` image. Tanka configurations that
...
+- The list of configuration files in the KV store and their associated hash values used for lookup in the ring.
```
Comment (clayton-cornell) [inline docs/sources/static/configuration/scraping-service.md:182]:
```suggestion
- The list of configuration files in the KV store and associated hash values used for lookup in the ring.
```

**Review by clayton-cornell — COMMENTED** [review]
Some doc review comments :-)

**Comment by clayton-cornell** [comment]
> I see some checks are failing but I have no idea what could be causing that. Is there anything I need to change?

That looks like a server connectivity hiccup. I think it'll pass the builds today.

**Review by clayton-cornell — APPROVED** [review]

**Pull request merged**